contrib/completion/bash/docker

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/env bash
+# shellcheck disable=SC2155
```

Comment:
Please add a comment where to find documentation on shellcheck warnings: `https://github.com/koalaman/shellcheck/wiki/SCXXXX`